inference-engine/ie_bridges/python/install/wheel/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2018-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set(WHEEL_PACKAGE_NAME "openvino" CACHE STRING "Name of the package")
+set(WHEEL_LICENCE_TYPE "Proprietary - Intel" CACHE STRING "License for the package")
+set(WHEEL_AUTOR "Intel Corporation" CACHE STRING "Package author’s name")
+set(WHEEL_AUTOR_EMAIL "openvino_pushbot@intel.com" CACHE STRING "Email address of the package author")
+set(WHEEL_DESC "Inference Engine Python* API" CACHE STRING "Short, summary description of the package")
```

Comment:
Should it be the OpenVINO package?
I thought that we should have one python package for all OpenVINO component.

Reply:
It's only IE - just runtime. For MO we need extra dependencies for conversion (TF, ONNX and so on) and it's better to keep separately.

Reply:
I would agree with @ilyachur  it is better to minimize wheel distros, I took this short description form Python API.
But full project description for OpenVino toolkit - https://pypi.org/project/openvino/

Reply:
Just for example, nobody need TensorFlow if they need to convert to convert PyTorch model. Why we force to install it?

Reply:
At the current moment Inference Engine Python API requires nGraph python API.
So I propose to have the one python wheel at least for Inference Engine and nGraph APIs.

Reply:
@ilyachur, seems like we just mean different things. I mean we need to separate MO and IE components.

Reply:
The latest commits fix this comment, the wheel package includes both ngraph and ie content